src/pages/About.mdx

```diff
@@ -14,48 +14,40 @@ import LinkExternal from '../components/Router/LinkExternal'
+This web application serves as an interactive planning tool for COVID-19 outbreaks in communities across the world.
+It implements a simple Susceptible-Infected-Recovered (SIR) model with additional categories for individuals exposed to the virus that are not yet infectious,
 severely sick people in need of hospitalization, people in critical condition, and a fatal category.
 
-The source code of this tool is freely available at <LinkExternal url="https://github.com/neherlab/covid19_scenarios">github.com/neherlab/covid19_scenarios</LinkExternal> and we welcome contributions in any form: comments, suggestions, help with development. Join our <LinkExternal url="https://github.com/neherlab/covid19_scenarios/discussions">General Discussion Thread</LinkExternal>.
+The source code of this tool is freely available at <LinkExternal url="https://github.com/neherlab/covid19_scenarios">github.com/neherlab/covid19_scenarios</LinkExternal> and we welcome contributions in any form: comments, suggestions, or help with development. We also welcome you to join our <LinkExternal url="https://github.com/neherlab/covid19_scenarios/discussions">General Discussion Thread</LinkExternal>.
 A preprint describing covid19-scenarios.org is now available on <LinkExternal url="https://www.medrxiv.org/content/10.1101/2020.05.05.20091363v2">medrxiv (doi: 10.1101/2020.05.05.20091363v2)</LinkExternal>.
 
 ## Basic assumptions
 
 The model works as follows:
-
- - susceptible individuals are exposed/infected through contact with infectious individuals. Each infectious individual causes on average $R_0$ secondary infections while they are infectious.
-Transmissibility of the virus could have seasonal variation which is parameterized with the parameter
-"seasonal forcing" (amplitude) and "peak month" (month of most efficient transmission).
-
- - exposed individuals progress to a symptomatic/infectious state after an average latency. This progression happends in three stages to ensure the distribution of times spend in the exposed compartment is more realistic than a simple exponential.
-
- - infectious individuals recover or progress to severe disease. The ratio of recovery to severe progression depends on age
-
- - severely sick individuals either recover or deteriorate and turn critical. Again, this depends on the age
-
- - critically ill individuals either get admitted to ICU (if space is available), or are placed in an overflow compartment. Younger age-groups are given preferential access to ICU.
-
- - critically ill individuals either return to regular hospital or die. Again, this depends on the age and on whether they receive intensive care or not.
-
+  * Susceptible individuals are exposed/infected through contact with infectious individuals. Each infectious individual causes on average $R_0$ secondary infections while they are infectious.
+    - Transmissibility of the virus could have seasonal varia
```

Comment:
This sentence seems to be cut short, compared to the old one

Reply:
You are right, my apologies. I have made changes to fix it; hopefully, it's resolved!
Cheers.

Reply:
@samanthahamilton Thanks!
It looks good to me. I will just let Richard @rneher to review and merge if he's okay with the changes.